### ISSUE TITLE
Make the "links" look more like a menu

### ISSUE THREAD

**Issue by Wingo5315** [WordPress/twentynineteen#32]
I've had a look at the mockups and the "menu" doesn't actually look like a menu, which can be a real problem for some users.

**Comment by bamadesigner** [comment]
Yes, this goes against the "Predictable" philosophy behind accessibility. https://www.w3.org/WAI/WCAG21/quickref/#predictable

But, ultimately, if you want to keep this design, then you can't remove the underline styles because it makes the menu impossible to differentiate from text for those who are color blind to blue.

The underline is the only thing in this design that makes it clear those items are links and therefore a menu. https://www.w3.org/WAI/WCAG21/quickref/#use-of-color

**Comment by kjellr** [comment]
Thanks for the feedback. I'll give this some thought this week, and report back with some options and notes. 👍

**Comment by marekhrabe** [comment]
I’d love if there was a choice between the current simple menu and something more standard, think twenty seventeen style. I can see usecases for both of them and it would be a pitty if this was a show-stopper for bigger sites that need more complex navigation.

**Comment by kjellr** [comment]
Thanks for the feedback, everyone. We've heard this feedback in a few places. I'd like to share an iteration that should hopefully help allay the concerns raised above, and also touch on some related issues raised in #33 and #1.

This thread doesn't have a screenshot of the current menu, so here it is for comparison's sake: 

![current-menu](https://user-images.githubusercontent.com/1202812/47303974-5e42b780-d5f3-11e8-9a87-2b966c45c9fd.png)

As expressed above, the core issue is that the menu is located seamlessly within the same content area as the site title and description. Because there are no line breaks or space separating the menu from the content, it may be difficult to find. It may also be difficult to differentiate the menu from the static text around it.

Here's my update:

_Default example_
![new](https://user-images.githubusercontent.com/1202812/47304355-5cc5bf00-d5f4-11e8-9e9c-9654e3e98b5e.png)

_Example with lots of menu items_
![many-items](https://user-images.githubusercontent.com/1202812/47304568-02792e00-d5f5-11e8-9313-52ea6955f64d.png)

_Small screen example_
<img width="379" alt="screen shot 2018-10-22 at 11 32 58 am" src="https://user-images.githubusercontent.com/1202812/47304572-04db8800-d5f5-11e8-9ae0-5fe3096e4b65.png">


This iteration: 

- Makes make the menu more obviously a menu by placing it on its own line all the time, and adding extra vertical space to differentiate it from the site title/description. It still exists in the same general location, but has its own dedicated space. 
- Removes the punctuation (`,` and `.`) from between menu items, for better i11n support.
- Uses a [priority+](https://css-tricks.com/the-priority-navigation-pattern/) navigation pattern to help account for cases where there  are very-long menus and menu items. This is also useful on mobile, where it  becomes a slightly more efficient version of the usual ["hamburger"](https://en.wikipedia.org/wiki/Hamburger_button) menu: rather than hide _all_ menu items by default, it allows users to select some items directly without needing to tap first into a menu. (Check out #143 for an example of how submenus and mobile menus would work in this case)
- ... while still maintaining the same general aesthetic and feel of the original version. 

I'm interested to hear everyone's thoughts — any ideas for further iteration or improvement?

**Comment by joyously** [comment]
That is _so_ much better!

**Comment by Wingo5315** [comment]
This looks a lot better! Thank you @kjellr!

**Comment by nielslange** [comment]
Happy to implement this later and create a PR. 😉

**Comment by kjellr** [comment]
🙌 Sounds great, thanks @nielslange!

**Comment by nielslange** [comment]
@kjellr In #263 I've implemented this issue partially. Unfortunately, I haven't been able to implement the priority+ navigation pattern as [mentioned by you](https://github.com/WordPress/twentynineteen/issues/32#issuecomment-431892606). While I found several CSS and JS based solutions, I wasn't able to get one of them working.To avoid going down the rabbit hole, I've wrapped up and created the clean PR #263.

It would be great if an JS expert could take over the implementation of the priority+ navigation patterm.

**Comment by kjellr** [comment]
Sounds good, thanks @nielslange — I've opened #268 to track the priority+ navigation work.

**Comment by nielslange** [comment]
Thanks, @kjellr!
